### ISSUE TITLE
No support for 64bit counters?

### ISSUE THREAD

**Issue by dragz** [ganglia/gmond_python_modules#246]
It does not seem to be possible to set up 64bit counters i python modules. type uint creates a 32bit rrd db, uint64 is not supported.

I'm trying to create an infiniband module and the kernel traffic stat counters are 64bit. (the 32bit counters overflows after two seconds) The plan was to use the counters as is with positive slope and let rrd do the rate calculations.

**Comment by vvuksan** [comment]
Unfortunately using slope with Ganglia is potentially challenging since you would have to set the update interval to the same as your gmetad updates e.g. every 15 seconds which is why in general I recommend you compute deltas

**Comment by dragz** [comment]
Yes, I took the pragmatic approach and just keep the state in a python shelve between calls. How about 64bit support? The uint type becomes uint32 in rrd. This is maybe not the right place, maybe a report to monitor-core is better?
